spec/requests/graphql/users_query_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "users query" do
+  context "when users exist" do
+    let!(:user) { create(:user) }
+
+    let(:user_data) do
+      {
+        "id" => user.id.to_s,
+        "email" => user.email
+      }
+    end
+
+    let(:query_string) do
+      "{
+        users {
+          id
+          email
+        }
+      }"
+    end
+
+    let(:request) { post graphql_path, params: { query: query_string } }
```

Comment:
Sounds good use subject?
```suggestion
    subject { post graphql_path, params: { query: query_string } }
```

Reply:
rubocop complains if we do it